README.md

```diff
@@ -201,7 +201,9 @@ The player controls can be customized with the following parameters:
 | `playPause`             |   `Boolean`    | `true`  | Display the play/pause button on the control bar     |
 | `progressBar`           |   `Boolean`    | `true`  | Display the progress bar on the control bar          |
 | `time`                  |   `Boolean`    | `true`  | Display the time information on the control bar      |
+| `seekSeconds`           |   `Number`     |  `5`    | Seconds to seek backward/forward with shortcuts      |
```

Comment:
I'd choose a parameter name without reference to the chosen unit, like `seekTime`.

For the description, `Customise the backward/forward seek time using shortcuts`

Reply:
For the name, it's fine.
For the description, I will say the chosen unit, something like: `Set seek time seconds of the backward and forward shortcuts`.

Reply:
Good for me